packages/atomic/src/components/ipx/atomic-ipx-button/atomic-ipx-button.tsx

```diff
@@ -99,7 +108,7 @@ export class AtomicIPXButton {
   }
 
   private get ipxModal() {
-    return document.querySelector('atomic-ipx-modal')!;
+    return this.bindings.interfaceElement.querySelector('atomic-ipx-modal')!;
```

Comment:
If this is something we do in the base modal & others as well, might be a good fix there too  🙏

Reply:
(For the use case of having multiple interfaces in 1 page)

Reply:
I see the use of document.querySelector in the following files: 

- [atomic-recs-list](https://github.com/coveo/ui-kit/blob/feat/SVCINT-2155/packages/atomic/src/components/recommendations/atomic-recs-list/atomic-recs-list.tsx#L236)
- [atomic-search-interface](https://github.com/coveo/ui-kit/blob/feat/SVCINT-2155/packages/atomic/src/components/search/atomic-search-interface/atomic-search-interface.tsx#L263-L264)
- [default-init](https://github.com/coveo/ui-kit/blob/feat/SVCINT-2155/packages/atomic/.storybook/default-init.ts#L19)
- [recs-init](https://github.com/coveo/ui-kit/blob/feat/SVCINT-2155/packages/atomic/.storybook/recommendation/recs-init.ts#L19)
- [atomic-external](https://github.com/coveo/ui-kit/blob/feat/SVCINT-2155/packages/atomic/src/components/search/atomic-external/atomic-external.tsx#L39)

I could make the changes there as well, but I feel these changes might have sneaky impacts and I think I would not be the best person to assess those impacts. If its alright with you, I'd like to just modify the behaviour of the ipx-button so that I can deploy this and use it in the SIS quickly 😅

Reply:
You can proceed with the change only to this bit for this PR I think, but if you can open a PR with the change in the other places, we would for sure appreciate 🙏

Reply:
Deal!
https://coveord.atlassian.net/browse/SVCINT-2159